Demo/Application/Features/SEPA Direct Debit/BraintreeDemoSEPADirectDebitViewController.swift

```diff
@@ -81,5 +81,65 @@ class BraintreeDemoSEPADirectDebitViewController: BraintreeDemoBaseViewControlle
     private func generateRandomCustomerID() -> String {
         String(UUID().uuidString.replacingOccurrences(of: "-", with: "").prefix(20))
     }
+
+    // MARK: - Helper functions for IBAN generation
+
+    private func generateRandomIBAN() -> String {
+        let countryCode: String = "FR"
+        let bankCode: String = "30006"
+        let branchCode: String = "00001"
```

Comment:
There is some weird stuff that needed to happen where we need the leading 0's in some cases and anything not of a String type (such as Int) will strip these.